packages/frontend/src/App.tsx

```diff
@@ -1,13 +1,69 @@
-import { hot } from "react-hot-loader"
-import * as React from "react"
-import "./style.css"
+import { hot } from 'react-hot-loader';
+import React, { useState, useEffect } from 'react';
+import './style.css';
+import axios from 'axios';
 
-const App = () => {
-  return(
-    <div className="title-box">
-      <h1 className="title">Ahoy there, Collectivists! Here be Dragons!! Yargh!</h1>
-    </div>
-  )
+interface Counter {
+  counter: number;
 }
 
-export default hot(module)(App)
\ No newline at end of file
+const App = () => {
+  const [counter, setCounter] = useState(0);
+
+  useEffect(() => {
+    (async () => {
+      await axios.get<Counter>('http://localhost:5000/counter').then(response => {
+        setCounter(response.data.counter);
+    });
+    })()
+  },[]);
+
+  function changeCounter(action: string) {
+    switch (action) {
+      case 'INCREMENT':
+        (async () => {
+          await axios.post<Counter>('http://localhost:5000/counter', {"change" : "INCREMENT"})
+        })()
+        setCounter(counter + 1);
```

Comment:
Here you're increasing the counter in state by one instead of displaying the response from the API. Should we get display the response from the API or display what's in the front-end only? 

I think we should still merge this PR but let's ask Monarch what's best practice. I am curious